### ISSUE TITLE
Computercraft not working.

### ISSUE THREAD

**Issue by prozacgod** [orbwoi/UniversalRemote#23]
Seems I can't use the computer interface with this tool.   Binding and then trying to use it from a distance causes it to flicker up on the screen and then quickly go away.

The computer does turn on, so it's seeing a right click event for the gui.

**Comment by orbwoi** [comment]
Yeah, both computercraft and open computers don't use the normal GUI functions. They are currently not supported and will probably not work correctly. I suppose I could blacklist them for now to prevent confusion? I would need to add some new internal features in order to try and get them working which I haven't had time to do yet.
